apps/ui/src/utils/metadatas.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  BioIcon,
+  DiscordIcon,
+  FarcasterIcon,
+  GithubIcon,
+  LocationIcon,
+  MailIcon,
+  TelegramIcon,
+  TwitterIcon,
+  UserIcon,
+  WebsiteIcon,
+} from '../components/icons/';
+
+export const Metadatas = {
+  General: [
+    {
+      key: 'nickname',
+      title: 'Nickname',
+      icon: <UserIcon w={7} h={7} />,
+      description: 'Your display name',
+      validated: true,
+    },
+    {
+      key: 'shortBio',
+      title: 'Short bio',
+      icon: <BioIcon w={7} h={7} />,
+      description: 'A brief description about you',
+      validated: null,
+    },
```

Comment:
```suggestion
  General: [
    {
      key: 'nickname',
      title: 'Nickname',
      icon: UserIcon,
      description: 'Your display name',
      validated: true,
    },
    {
      key: 'shortBio',
      title: 'Short bio',
      icon: BioIcon,
      description: 'A brief description about you',
      validated: null,
    },
```

Reply:
apply the same to the other icons. In the place where you use the icon, you can instantiate it like this:

```ts
const [nickname] = Metadatas.General[0];

<nickname.icon w={7} h={7} />
```